io.openems.edge.evse.chargepoint.alpitronic/src/io/openems/edge/evcs/alpitronic/EvcsAlpitronicImpl.java

```diff
@@ -74,6 +82,15 @@ public class EvcsAlpitronicImpl extends AbstractOpenemsModbusComponent
 	/** Modbus offset for multiple connectors. */
 	private final IntFunction<Integer> offset = addr -> addr + this.config.connector().modbusOffset;
 
+	/** Software version for register mapping compatibility. */
+	private SemanticVersion firmwareVersion = null;
+	private Integer versionMajor = null;
+	private Integer versionMinor = null;
+	private Integer versionPatch = null;
```

Comment:
This is redundant. I believe none is actually required, because we also have a Channel.